test/paper-input.html

```diff
@@ -162,6 +162,37 @@
 
     });
 
+    suite('focus/blur events', function() {
+      var input;
+
+      setup(function() {
+        input = fixture('basic');
+      });
+
+      test('focus/blur events fired on host element', function(done) {
```

Comment:
I want to add the test since `IronControlState` is a later addition to this element. For `<paper-textarea>`, since it's redispatching the events twice it's slightly different than the test case in `IronControlState`.